templates/dart/lib/role.dart.twig

```diff
@@ -0,0 +1,26 @@
+part of {{ language.params.packageName }};
+
+class Role {
+    static String any() {
+        return 'any';
+    }
+
+    static String user(String id) {
+        return 'user:$id';
+    }
+
+    static String users() {
+        return 'users';
+    }
+
+    static String guests() {
+        return 'guests';
+    }
+
+    static String team(String id, [String? role]) {
+        if(role != null) {
```

Comment:
I think this code will fail if role is passed as an empty string.. 

```dart
Role::team('123', ''); // Will return team:123/
```

Let's be consistent with how we handle this logic. 

Please refer to the Kotlin comment. We use an empty string as a default parameter and then use the language built in `isEmpty()` function for this check

Reply:
done 👍